docs/index.md

```diff
@@ -13,36 +13,34 @@ Pydantic is the most widely used data validation library for Python.
 
 Fast and extensible, Pydantic plays nicely with your linters/IDE/brain. Define how data should be in pure, canonical Python 3.8+; validate it with Pydantic.
 
-!!! success "Migrating to Pydantic V2"
-    Using Pydantic V1? See the [Migration Guide](migration.md) for notes on upgrading to Pydantic V2 in your applications!
-
-```py title="Pydantic Example" requires="3.10"
-from datetime import datetime
-from typing import Tuple
-
-from pydantic import BaseModel
+!!! logfire "Monitor Pydantic with Logfire :rocket:"
+    Built by the same team as Pydantic, [Logfire](https://pydantic.dev/logfire) is an application monitoring tool that is as simple to use and powerful as Pydantic itself.
 
+    Logfire integrates with many popular Python libraries including FastAPI, OpenAI and Pydantic itself, so you can use pydantic to monitor Pydantic validations and understand why some inputs fail validation:
 
-class Delivery(BaseModel):
-    timestamp: datetime
-    dimensions: Tuple[int, int]
+    ```py title="Monitoring Pydantic with Logfire"
+    from pydantic import BaseModel
+    import logfire
 
+    logfire.configure()
+    logfire.instrument_pydantic() # record all Pydantic validations,
 
-m = Delivery(timestamp='2020-01-02T03:04:05Z', dimensions=['10', '20'])
-print(repr(m.timestamp))
-#> datetime.datetime(2020, 1, 2, 3, 4, 5, tzinfo=TzInfo(UTC))
-print(m.dimensions)
-#> (10, 20)
-```
-
-!!! question "Why is Pydantic named the way it is?"
+    class Delivery(BaseModel):
+        timestamp: datetime
+        dimensions: tuple[int, int]
 
-    The name "Pydantic" is a portmanteau of "Py" and "pedantic." The "Py" part indicates that the library is associated with Python, and
-    "pedantic" refers to the library's meticulous approach to data validation and type enforcement.
+    # this will record details of a successful validation to logfire
+    Delivery(timestamp='2020-01-02T03:04:05Z', dimensions=['10', '20'])
+    print(repr(m.timestamp))
+    #> datetime.datetime(2020, 1, 2, 3, 4, 5, tzinfo=TzInfo(UTC))
+    print(m.dimensions)
+    #> (10, 20)
 
-    Combining these elements, "Pydantic" describes our Python library that provides detail-oriented, rigorous data validation.
+    # this will record details of a failed validation to logfire
+    Delivery(timestamp='2020-01-02T03:04:05Z', dimensions=['10', '20', '30'])
+    ```
+    [Learn more about Pydantic Logfire](https://pydantic.dev/logfire)
```

Comment:
```suggestion
    [Learn more about Pydantic Logfire](https://pydantic.dev/logfire), including examples of how to instrument more complex apps.
```
This feels like an awkward thing to say but basically I am trying to allude to the point that, it's not just good for instrumenting toy examples, which is all this is (and all it's intended to be). Maybe we just explicitly make that point — 
```suggestion
    This is just a toy example, but hopefully makes clear the potential value of instrumenting a more complex application.
    
    [Learn more about Pydantic Logfire](https://pydantic.dev/logfire)
```
(or similar)